src/Destructurama.Attributed/Attributed/LogReplacedAttribute.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright 2015-2020 Destructurama Contributors, Serilog Contributors
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Text.RegularExpressions;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Destructurama.Attributed
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public class LogReplacedAttribute : Attribute, IPropertyDestructuringAttribute
+    {
+        private readonly string pattern;
```

Comment:
Nit: project doesn't use explicit `private` or `internal`, and uses `_field` underscored field names.